pvr.octonet/resources/language/resource.language.en_gb/strings.po

```diff
@@ -27,3 +27,33 @@ msgstr ""
 msgctxt "#30001"
 msgid "Could not load chanellist"
 msgstr ""
+
+msgctxt "#30002"
+msgid "Open stream using"
+msgstr ""
+
+msgctxt "#30003"
+msgid "Live stream API"
```

Comment:
What is live stream api? Can we find better wording?

Reply:
Done